src/main.js

```
@@ -0,0 +1,19 @@
+function getDaysInMonth(month, year) {
+  try {
+    if (!Number.isInteger(year) && Number.isInteger(month) && month <= 12) {
+      throw new Error('incorrrect');
+    };
+
+    let daysInMonth = 33 - new Date(year, month - 1, 33).getDate();
+    return (daysInMonth);
+  } catch (err) {
+      return err.message;
```

`... the function should throw an exception.`